[2023-02-16T07:49:46.381+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: dag_e2e_etl_v02.t1a_read_csv_from_web manual__2023-02-16T07:49:44.325973+00:00 [queued]>
[2023-02-16T07:49:46.389+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: dag_e2e_etl_v02.t1a_read_csv_from_web manual__2023-02-16T07:49:44.325973+00:00 [queued]>
[2023-02-16T07:49:46.390+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-02-16T07:49:46.391+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-02-16T07:49:46.391+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-02-16T07:49:46.416+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): t1a_read_csv_from_web> on 2023-02-16 07:49:44.325973+00:00
[2023-02-16T07:49:46.431+0000] {standard_task_runner.py:55} INFO - Started process 34025 to run task
[2023-02-16T07:49:46.438+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_e2e_etl_v02', 't1a_read_csv_from_web', 'manual__2023-02-16T07:49:44.325973+00:00', '--job-id', '146', '--raw', '--subdir', 'DAGS_FOLDER/dag_e2e_etl.py', '--cfg-path', '/tmp/tmpnh8hpvyl']
[2023-02-16T07:49:46.443+0000] {standard_task_runner.py:83} INFO - Job 146: Subtask t1a_read_csv_from_web
[2023-02-16T07:49:46.555+0000] {task_command.py:388} INFO - Running <TaskInstance: dag_e2e_etl_v02.t1a_read_csv_from_web manual__2023-02-16T07:49:44.325973+00:00 [running]> on host 20eb5e5814e6
[2023-02-16T07:49:47.502+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag_e2e_etl_v02
AIRFLOW_CTX_TASK_ID=t1a_read_csv_from_web
AIRFLOW_CTX_EXECUTION_DATE=2023-02-16T07:49:44.325973+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-02-16T07:49:44.325973+00:00
[2023-02-16T07:49:48.810+0000] {python.py:177} INFO - Done. Returned value was:      Survived  Pclass  ... Parents/Children Aboard     Fare
0           0       3  ...                       0   7.2500
1           1       1  ...                       0  71.2833
2           1       3  ...                       0   7.9250
3           1       1  ...                       0  53.1000
4           0       3  ...                       0   8.0500
..        ...     ...  ...                     ...      ...
882         0       2  ...                       0  13.0000
883         1       1  ...                       0  30.0000
884         0       3  ...                       2  23.4500
885         1       1  ...                       0  30.0000
886         0       3  ...                       0   7.7500

[887 rows x 8 columns]
[2023-02-16T07:49:48.969+0000] {xcom.py:635} ERROR - Object of type DataFrame is not JSON serializable. If you are using pickle instead of JSON for XCom, then you need to enable pickle support for XCom in your *** config or make sure to decorate your object with attr.
[2023-02-16T07:49:48.987+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 72, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 2297, in xcom_push
    session=session,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 72, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/xcom.py", line 240, in set
    map_index=map_index,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/xcom.py", line 627, in serialize_value
    return json.dumps(value, cls=XComEncoder).encode("UTF-8")
  File "/usr/local/lib/python3.7/json/__init__.py", line 238, in dumps
    **kw).encode(obj)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/json.py", line 176, in encode
    return super().encode(o)
  File "/usr/local/lib/python3.7/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/local/lib/python3.7/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/json.py", line 170, in default
    return super().default(o)
  File "/usr/local/lib/python3.7/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type DataFrame is not JSON serializable
[2023-02-16T07:49:49.078+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=dag_e2e_etl_v02, task_id=t1a_read_csv_from_web, execution_date=20230216T074944, start_date=20230216T074946, end_date=20230216T074949
[2023-02-16T07:49:49.127+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 146 for task t1a_read_csv_from_web (Object of type DataFrame is not JSON serializable; 34025)
[2023-02-16T07:49:49.272+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2023-02-16T07:49:49.522+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
